modules/Lending.py

```diff
@@ -201,6 +201,7 @@ def cancel_all():
 
 
 def lend_all():
+    print("Rate : {0}".format(Analysis.get_rate_suggestion('BTC')))
```

Comment:
Yea, there's probably a few more in there. It's useful to get the rates that out of the module without actually making a loan. 

They'll probably stay until it's suitable for merging to master. I should probably put 'do not merge' on this PR actually.